tests/Behat/Context/Setup/ChannelContext.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Behat\Context\Setup;
+
+
+use Behat\Behat\Context\Context;
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Formatter\StringInflector;
+use Sylius\Component\Core\Test\Services\DefaultChannelFactoryInterface;
+
+
+final class ChannelContext implements Context
+{
+    /**
+     * @var SharedStorageInterface
+     */
+    private $sharedStorage;
+
+    /**
+     * @var DefaultChannelFactoryInterface
+     */
+    private $unitedStatesChannelFactory;
+
+    /**
+     * @var DefaultChannelFactoryInterface
+     */
+    private $defaultChannelFactory;
+
+    /**
+     * @param SharedStorageInterface $sharedStorage
+     * @param DefaultChannelFactoryInterface $unitedStatesChannelFactory
+     * @param DefaultChannelFactoryInterface $defaultChannelFactory
+     * @param ChannelRepositoryInterface $channelRepository
+     * @param ObjectManager $channelManager
+     */
+    public function __construct(
+        SharedStorageInterface $sharedStorage,
+        DefaultChannelFactoryInterface $unitedStatesChannelFactory,
+        DefaultChannelFactoryInterface $defaultChannelFactory
+    ) {
+        $this->sharedStorage = $sharedStorage;
+        $this->unitedStatesChannelFactory = $unitedStatesChannelFactory;
+        $this->defaultChannelFactory = $defaultChannelFactory;
+    }
+
+    /**
+     * @Given the store operates on a single green channel in "United States"
+     */
+    public function storeOperatesOnASingleColorChannelInUnitedStates()
+    {
+        $defaultData = $this->unitedStatesChannelFactory->create();
+        $defaultData->setColor('green');
```

Comment:
And we should get the channel from returned data: `$defaultData['channel']->setColor($color);`